pom.xml

```diff
@@ -50,6 +51,27 @@
             <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
             <version>2.3.0</version>
         </dependency>
+        <dependency>
```

Comment:
You should remove all these dependencies,
Pom.xml should have only this dependencies for test cases. I explained you last time and shared some links 
<dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-test</artifactId>
            <scope>test</scope>

Reply:
if i remove this dependency springdoc-openapi-starter-webmvc-ui then i will get an error from swagger.

Reply:
No, I am talking about <groupId>junit</groupId>

Reply:
deleted